### PR TITLE
Create schema

### PR DESCRIPTION
**Pull request by tiffanyjalovec**
MVP with airbnb styling

### CODE REVIEW

File: database/models.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const db = require('./index.js');
+
+module.exports = {
+  gather: (restId, callback) => {
+    const queryStr = `SELECT * FROM photos WHERE restaurant_id = ${restId}`;
```
Comment (Try-Try-Again) [inline database/models.js:5]: Is this safe from SQL injection?

File: database/seed.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 const generate = () => {
-  for (let i = 0; i < 10; i += 1) {
-    for (let j = 0; j < pics.length; j += 1) {
-      db.query(`INSERT INTO photos (image, restaurant_id) VALUES ( '${pics[j]}' , ${i})`, (err, result) => {
+  for (let i = 1; i < 100; i += 1) {
+    for (let j = 1; j < 20; j += 1) {
+      const randomNumber = Math.floor(Math.random() * Math.floor(50));
+      const url = `https://dibsondinner.s3-us-east-2.amazonaws.com/dibsondinner/photo-${randomNumber}.jpg`;
+      db.query(`INSERT INTO photos (image, restaurant_id) VALUES ( '${url}', ${i})`, (err, result) => {
```
Comment (Try-Try-Again) [inline database/seed.js:8]: Another possible risk. I'm not sure though. I'm new too :)

**Review by Try-Try-Again — APPROVED** [review]
Possible SQL injection risk. But I like living on the edge so... LGTM :D

**Review by tombohere — COMMENTED** [review]
code looks clean and functional

File: database/models.js (hunk 1 above)
Comment (tiffanyjalovec) [reply database/models.js:5]: I will check into it! Thanks!!

**Pull request merged**